app/models/area.rb

```diff
@@ -4,12 +4,14 @@ class Area < ActiveRecord::Base
 
   class << self
     def for_code(area_code)
+      return nil unless area_code.present?
```

Comment:
That's true, yeah. I was thinking that putting `nil` would make ti more explicit, but since we are all familiar with that I'm fine taking it out (and it's slightly cleaner without it).